openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.less

```diff
@@ -37,7 +37,7 @@
     top: 0;
     margin-left: -8px;
     margin-right: -8px;
-    z-index: 10;
+    z-index: 300;
```

Comment:
We will avoid this random z-index. Please make note of it. We will remove all the random z-index